internal/provider/provider.go

```diff
@@ -108,7 +109,7 @@ func providerConfigure(_ context.Context, data *schema.ResourceData) (interface{
 		password, isSetPassword := data.GetOk("sddc_manager_password")
 		hostName, isSetHost := data.GetOk("sddc_manager_host")
 		if !isVcfUsernameSet || !isSetPassword || !isSetHost {
-			return nil, diag.Errorf("SDDC Manager username, password and host must be provided")
+			return nil, diag.Errorf("error: SDDC Manager username, password, and host must be provided")
```

Comment:
"error: " is redundant as a Error struct is returned which the terraform SDK will log at error level.

Reply:
Resolved in e8f6082.